src/modules/battery.cpp

```diff
@@ -19,11 +26,12 @@ waybar::modules::Battery::Battery(const std::string& id, const Json::Value& conf
   if (global_watch < 0) {
     throw std::runtime_error("Could not watch for battery plug/unplug");
   }
-
+#endif
   worker();
 }
 
 waybar::modules::Battery::~Battery() {
+#if (__Linux__)
```

Comment:
Witthout `defined` may trigger `-Wundef` on FreeBSD and should be lowercase:
```suggestion
#if defined(__linux__)
```